### PR TITLE
Added `dry-run` to `README` example

### PR DESCRIPTION
**Pull request by dowusu-antwi**
Towards fixing https://github.com/SupertechLabs/general-superstaq/issues/52.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ service = css.Service(
     api_key="""Insert superstaq token that you received from https://superstaq.super.tech""",
     verbose=True,
 )
-job = service.create_job(circuit=circuit, repetitions=1, target="ibmq_qasm_simulator")
+job = service.create_job(circuit=circuit, repetitions=1, target="ibmq_qasm_simulator", method="dry-run")
```
Comment (vtomole) [inline README.md:32]: Let's add a sentence or two explaining this `dry-run` flag.

Comment (dowusu-antwi) [reply README.md:34]: Done.

File: README.md [hunk 2]
```diff
@@ -29,6 +29,8 @@ service = css.Service(
     api_key="""Insert superstaq token that you received from https://superstaq.super.tech""",
     verbose=True,
 )
+
+# Submitting a circuit to "ibmq_qasm_simulator". Providing the "dry-run" method parameter instructs SuperstaQ to simulate the circuit, ans is available to free trial users.
```
Comment (vtomole) [inline README.md]:
```suggestion
# Submitting a circuit to "ibmq_qasm_simulator". Providing the "dry-run" method parameter instructs SuperstaQ to simulate the circuit, and is available to free trial users.
```

**Review by vtomole — APPROVED** [review]
LGTM

**Pull request merged**